src/meta/model_v2/src/function.rs

```diff
@@ -44,6 +44,7 @@ pub struct Model {
     pub link: Option<String>,
     pub identifier: Option<String>,
     pub body: Option<String>,
+    pub compressed_binary: Option<Vec<u8>>,
```

Comment:
Do we have an experience value that how large it can be for a WASM binary? I'm concerned about the possibility of running out of memory as all catalogs are cached in memory.

Reply:
It is about 300KB after compression for our wasm binary in e2e test.